editors/neovim/files/patch-nlua0-includes.diff

```diff
@@ -0,0 +1,12 @@
+diff --git src/nvim/CMakeLists.txt src/nvim/CMakeLists.txt
+index a100e733d..9c601819b 100644
+--- src/nvim/CMakeLists.txt
++++ src/nvim/CMakeLists.txt
+@@ -66,6 +66,7 @@ else()
+   target_include_directories(main_lib SYSTEM BEFORE INTERFACE ${LUAJIT_INCLUDE_DIR})
+   target_link_libraries(main_lib INTERFACE ${LUAJIT_LIBRARY})
+   target_include_directories(nlua0 SYSTEM BEFORE PUBLIC ${LUAJIT_INCLUDE_DIR})
++  target_include_directories(nlua0 AFTER PUBLIC /opt/local/include/lua5.1)
```

Comment:
don't hardcode "/opt/local" as user might install in a different ${prefix}. Not sure if there is a suitable CMake variable already available to use here. Otherwise the typical solution is to patch in a placeholder and then do a "reinplace" to put in "${prefix}" in the "post-patch" phase. There are many examples of ports doing similar thing you can check for inspiration.